rules/windows/process_creation/proc_creation_win_schtasks_system.yml

```diff
@@ -0,0 +1,38 @@
+title: Schtasks Creation Or Modification With SYSTEM Privileges
+id: 89ca78fd-b37c-4310-b3d3-81a023f83936
+status: experimental
+description: Detects the creation or update of a scheduled task to run with "NT AUTHORITY\SYSTEM" privileges
+author: Nasreddine Bencherchali
+references:
+    - https://www.elastic.co/security-labs/exploring-the-qbot-attack-pattern
+date: 2022/07/28
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection_root:
+        Image|endswith: '\schtasks.exe'
+        CommandLine|contains:
+            - ' /change '
+            - ' /create '
+    selection_run:
+        CommandLine|contains: '/ru '
+    selection_user:
+        CommandLine|contains:
+            - 'NT AUT'
+            - ' SYSTEM '
```

Comment:
' SYSTEM ' more '\SYSTEM' ?

Reply:
Let's use our good old:
```
        User|contains: 
            - 'AUTHORI'
            - 'AUTORI'
```

Reply:
"SYSTEM" is a valid value for "schtasks" which is equivalent to "NT AUTHORITY\SYSTEM" and I didn't want to use "AUTORI" in the command-line as it contains folders and username and "AUTORI" cannot be as specific in my opinion